### ISSUE TITLE
Python test script UploadZipAndDeleteList.py is broken

### ISSUE THREAD

**Issue by tommydeboer** [molgenis/molgenis#7666]
### How to Reproduce
1. Download and extract https://drive.google.com/open?id=0B3D6LFtvxTara0tIN0ZhTjEwODQ
2. Run `wget https://<molgenis>/molgenis.py` in the same folder 
3. Run `UploadZipAndDeleteList.py` 

### Expected behavior
The script runs without errors

### Observed behavior
```
Traceback (most recent call last):
  File "UploadZipAndDeleteList.py", line 5, in <module>
    response = session.upload_zip("Archive.zip").split('/')
  File "/Users/tommy/temp/PythonAPIUploadZipAndDeleteList/molgenis.py", line 199, in upload_zip
    return response.json()
  File "/usr/local/lib/python3.5/site-packages/requests/models.py", line 841, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/local/Cellar/python3/3.5.2_3/Frameworks/Python.framework/Versions/3.5/lib/python3.5/json/__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "/usr/local/Cellar/python3/3.5.2_3/Frameworks/Python.framework/Versions/3.5/lib/python3.5/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/Cellar/python3/3.5.2_3/Frameworks/Python.framework/Versions/3.5/lib/python3.5/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by LuukDijkhuis** [comment]
Please put this into the CI

**Comment by LuukDijkhuis** [comment]
done

**Comment by marikaris** [comment]
This issue is only fixed in python 2.7, it still fails in python 3
